README.md

```diff
@@ -1,7 +1,5 @@
 ## Book Reading List
 
-Basic python book reading list application. For practicing teamwork and GitHub collaboration. 
-
-Uses SQLite3 database to store data. 
-
-Requires at least Python 3.7.
+This is a basic reading list application. Users can add books, search books, see unread/read books, see all
```

Comment:
this should be in a different branch - be careful you check out main before making a new branch